2016/presentations/Rozklady_odwrotnie_potegowe/text.md

```diff
@@ -0,0 +1,34 @@
+# Co mają wspólnego trzęsienia ziemia, cyklon tropikalny, lawiny błotne i nasze repozytoria? - Paweł Kopka
+Co łączy repozytoria, trzęsienia ziemi, lawiny błotne oraz cyklony tropikalne? Pewnie większości z was przychodzi do głowy słowo "katastrofa", ale dzisiaj nie o tym. Odpowiedź na otwierające pytanie to: rozkłady odwrotnie potęgowe. Nie są tak "popularne" jak rozkład normalny, ale też wypełnianiają znaczącą przestrzeń w naszym świecie.
+## Rozkład odwrotnie potęgowy
+Rozkłady odwrotnie potęgowe, czasem znane jako rozkład Pareto lub Riemann Zeta, można opisać dość prostym wzorem: f(x) = c/x^a. Oczywiście charakteryzują się one pewnymi założeniami (takimi jak x nie może równać się zero), ale nie warto się w nie zagłębiać. W wielkim skrócie ten rozkład mówi, że małe zdarzenia zdarzają się znacznie częściej niż wielkie. Świetnym przykładem jest badanie Pareto, który interesował się zamożnością ludzi. Opisywał rozkład dóbr wśród społeczeństwa, a, jak można się domyślać, dużo więcej ludzi posiada mały dobytek, natomiast maleńka cześć społeczeństwa posiada dużo dłuższe ciągi zer na kontach. Jako ciekawostkę można dodać, że z tych badań powstała zasada Pareto, która mówi, że 20% populacji posiada 80% bogactwa. To nam mówi, jaka jest skala różnicy między ilością dóbr na początku rozkładu a w jego ogonie.
```

Comment:
Przecinek po "mówi" w ostatnim zdaniu niepotrzebny (to nie jest zdanie złożone porównawcze), natomiast przydałby się przed "a".